### PR TITLE
Backend BOARDS Create, Read

### PR DESCRIPTION
**Pull request by danguai**

### CODE REVIEW

**Review by jonevanmoore — DISMISSED** [review]

**Review by MasonKogami — APPROVED** [review]

File: migrations/versions/20220502_171842_boards_table.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('boards',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('user_id', sa.Integer(), nullable=False),
+    sa.Column('title', sa.String(length=30), nullable=False),
+    sa.Column('avatar_id', sa.Integer(), nullable=False),
+    sa.Column('created_at', sa.DateTime(timezone=True), server_default=sa.text('now()'), nullable=True),
+    sa.Column('updated_at', sa.DateTime(timezone=True), nullable=True),
```
Comment (Nick-Yawn) [inline migrations/versions/20220502_171842_boards_table.py:27]: I think this line needs an onupdate flag

**Review by Nick-Yawn — DISMISSED** [review]

File: migrations/versions/20220502_171842_boards_table.py (hunk 1 above)
Comment (Nick-Yawn) [reply migrations/versions/20220502_171842_boards_table.py:27]:
```
time_updated = Column(DateTime(timezone=True), onupdate=func.now())
```

**Review by Nick-Yawn — APPROVED** [review]

**Review by jonevanmoore — APPROVED** [review]

**Review by Nick-Yawn — CHANGES_REQUESTED** [review]
boards FK

**Review by Nick-Yawn — APPROVED** [review]

**Review by Nick-Yawn — CHANGES_REQUESTED** [review]
merge main

**Review by Nick-Yawn — APPROVED** [review]

**Pull request merged**